internal/provider/resource_password_test.go

```diff
@@ -365,6 +382,8 @@ func TestAccResourcePassword_Import_FromVersion3_1_3(t *testing.T) {
 				ImportState:        true,
 				ImportStateId:      "Z=:cbrJE?Ltg",
 				ImportStatePersist: true,
+				// TODO: Import state checks haven't been implemented in terraform-plugin-testing yet, so can't use value comparers for now
+				// TODO: Create import state check issue in terraform-plugin-testing
```

Comment:
We'll need to introduce support for state checks in import to update these tests, perhaps a miss on our original implementation, but I don't think it should be too difficult to introduce 🤞🏻

Reply:
Opened an issue and will update the TODO comments: https://github.com/hashicorp/terraform-plugin-testing/issues/365